src/hotspot/share/gc/shared/pretouchTask.cpp

```diff
@@ -67,6 +68,9 @@ void PretouchTask::pretouch(const char* task_name, char* start_address, char* en
   // Chunk size should be at least (unmodified) page size as using multiple threads
   // pretouch on a single page can decrease performance.
   size_t chunk_size = MAX2(PretouchTask::chunk_size(), page_size);
+  // Page-align the chunk size, so if start_address is also page-aligned (as
+  // is common) then there won't be any pages shared by multiple chunks.
+  chunk_size = align_down(chunk_size, page_size);
```

Comment:
This looks like a place where align_down_bounded could be used. Something like this:
size_t chunk_size = align_down_bounded(PretouchTask::chunk_size(), page_size);

Reply:
I either forgot about or never noticed that function.  Changed to use it.